src/main/java/com/couchbase/lite/support/JsonDocument.java

```diff
@@ -36,7 +36,13 @@ public Object jsonObject() {
                 tmp = new LazyJsonArray<Object>(json);
             } else {
                 try {
-                    tmp = Manager.getObjectMapper().readValue(json, Object.class);
+                    // this is for Jackson 2.5.0
```

Comment:
- Added comment, and pushed it.
- As byte[] has length property, it does not require NULL stop like C char array. So It seems Jackson 2.5.0 treats '\0' as part of data, but it is not able to handle it. I believe `Cursor.getBlob()` does not need to added '\0' at end of byte[], also Jackson 2.5.0 should handle it.